### PR TITLE
Support for RHEL 7 mariadb and issue #95

### PR DESCRIPTION
**Pull request by dsglaser**
Added support for RHEL 7 mariadb. Unfortunately it makes a change to tasks/main.yml to support it. 

Fixed issue #95 by making sure that datadir is created and selinux context set on the directory. 

Feel free to use or reject as needed. These were done to support our enterprise installations where I work

### CODE REVIEW

File: tasks/main.yml [hunk 1]
```diff
@@ -2,6 +2,17 @@
 # Include variables and define needed variables.
 - name: Include OS-specific variables.
   include_vars: "{{ ansible_os_family }}.yml"
+  when: ansible_os_family  != "RedHat"
+
+# Include RedHat 6 variables (mysql)
+- name: Include OS-specific variables.
+  include_vars: "{{ ansible_os_family }}-{{ ansible_lsb.major_release }}.yml"
```
Comment (geerlingguy) [inline tasks/main.yml]: If `ansible_lsb.major_release` just outputs `6` or `7`, no need for two tasks, we can just do one task without a `when` condition.

File: vars/RedHat-7.yml [hunk 2]
```diff
@@ -0,0 +1,15 @@
+---
+__mysql_daemon: mariadb
+__mysql_packages:
```
Comment (geerlingguy) [inline vars/RedHat-7.yml]: I like this approach! No need to have people manually override just for RHEL 7.

**Comment by geerlingguy** [comment]
I like this approach, but I have a couple comments for review. Thanks!

File: tasks/main.yml (hunk 1 above)
Comment (dsglaser) [inline tasks/main.yml]: Looks good to me.

**Comment by geerlingguy** [comment]
I based https://github.com/geerlingguy/ansible-role-mysql/pull/116 off this PR, and merged. Thanks again!

**Pull request merged**